plugin.go

```diff
@@ -14,6 +14,7 @@ func (e *ExtraClausePlugin) Name() string {
 func (e *ExtraClausePlugin) Initialize(db *gorm.DB) error {
 	db.Callback().Query().Clauses = []string{"WITH", "SELECT", "FROM", "WHERE", "GROUP BY", "UNION", "ORDER BY", "LIMIT", "FOR"}
 	db.Callback().Row().Clauses = []string{"WITH", "SELECT", "FROM", "WHERE", "GROUP BY", "UNION", "ORDER BY", "LIMIT", "FOR"}
+	db.Callback().Update().Clauses = []string{"WITH", "UPDATE", "SET", "FROM", "WHERE"}
```

Comment:
If `FROM` clause is exists in clauses, GORM force using `FROM` clause. So this code is not working in MySQL.
Please reuse clauses of `Dialector`.

```go
db.Callback().Update().Clauses = append([]string{"WITH"}, db.Callback().Update().Clauses...)
```